### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by thelovekesh**
Fixed some common typing mistakes. Removed one "from" from the Downloading a Release from GitHub topic.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,9 +31,9 @@ Available Commands:
   completion      Modify your shell so doctl commands autocomplete with TAB
   compute         Display commands that manage infrastructure
   databases       Display commands that manage databases
-  help            Help about any command
+  help            Help with any command
   invoice         Display commands for retrieving invoices for your account
-  kubernetes      Displays commands to manage Kubernetes clusters and configurations
+  Kubernetes      Displays commands to manage Kubernetes clusters and configurations
```
Comment (andrewsomething) [inline README.md:36]: Here `kubernetes` is a `doctl` sub-command. So it should remain lower cased.

**Review by andrewsomething — COMMENTED** [review]
Thanks for these improvements! Once you address the one comment I made in-line, I'll merge it.

File: README.md (hunk 1 above)
Comment (thelovekesh) [reply README.md]: Thank you for your response!  I have made changes as you suggested.

**Review by andrewsomething — APPROVED** [review]
:+1:  LGTM

Thanks again for the contribution!

**Comment by thelovekesh** [comment]
I just want to ask one thing.  Am I a contributor now to doctl? 
I am a student and learning new things.

**Comment by andrewsomething** [comment]
:smile: I'd say so!

image

**Pull request merged**